client/src/components/CardItem/index.js

```diff
@@ -0,0 +1,3 @@
+import CardItem from "./CardItem";
```

Comment:
Please, remove this file and transfer your re-export to /components/index.js file.